praw/models/reddit/emoji.py

```diff
@@ -78,6 +81,45 @@ def delete(self):
         )
         self._reddit.request("DELETE", url)
 
+    def update(
+        self,
+        post_flair=None,
+        user_flair=None,
+        mod_only=None,
+    ):
+        """Update the permissions of an emoji in this subreddit.
+
+        :param post_flair: (boolean) Whether the emoji appears in post flair.
+            Respects pre-existing settings if not provided.
+        :param user_flair: (boolean) Whether the emoji appears in user flair.
+            Respects pre-existing settings if not provided.
+        :param mod_only:  (boolean) Whether the emoji will be available for mod
+            use only. Respects pre-existing settings if not provided.
+
+        To make the emoji ``'test'`` available for mod use only in subreddit
+        ``'wowemoji'``, try:
+
+        .. code:: python
+
+           reddit.subreddit('wowemoji').emoji['test'].update(mod_only=True)
+        """
+        data = {
+            "name": self.name,
+            "post_flair_allowed": self.post_flair_allowed,
+            "user_flair_allowed": self.user_flair_allowed,
+            "mod_flair_only": self.mod_flair_only,
+        }
+
+        if isinstance(post_flair, bool):
+            data['post_flair_allowed'] = post_flair
+        if isinstance(user_flair, bool):
+            data['user_flair_allowed'] = user_flair
+        if isinstance(mod_only, bool):
+            data['mod_flair_only'] = mod_only
```

Comment:
For all three of these checks, I'd like to replace the `isinstance` call with `thing is not None`. This more cleanly captures the logic that we set the values that are passed as parameters.

Reply:
10-4